README.md

```diff
@@ -69,6 +71,217 @@ dd-handler recipe. The known workaround is to update your dependency to `chef_ha
 the `chef_handler` cookbook which is now shipped as a resource in Chef 14.
 Unfortunately, it will display a deprecation message to Chef 14 and 15 users.
 
+Upgrading to version 4.x of the cookbook
+========================================
+
+Some attributes have changed their names from version 3.x to 4.x of this cookbook. Use this reference table to update your configuration:
+
+| Action | Cookbook 3.x  | Cookbook 4.x  |
+|---|---|---|
+| Install Agent 7.x | Not supported |  `'agent_major_version' => 7` |
+| Install Agent 6.x | `'agent6' => true`  |  `'agent_major_version' => 6` |
+| Install Agent 5.x | `'agent6' => false`  |  `'agent_major_version' => 5` |
+| Pin agent version | `'agent_version'` or `'agent6_version'`  |  `'agent_version'` for all versions |
+| Change package_action | `'agent_package_action'` or `'agent6_package_action'`  |  `'agent_package_action'` for all versions |
+| Change APT repo | `'aptrepo'` or `'agent6_aptrepo'`  |  `'aptrepo'` for all versions |
+| Change APT dist | `'aptrepo_dist'` or `'agent6_aptrepo_dist'`  |  `'aptrepo_dist'` for all versions |
+| Change YUM repo | `'yumrepo'` or `'agent6_yumrepo'`  |  `'yumrepo'` for all versions |
+| Change SUSE repo | `'yumrepo_suse'` or `'agent6_yumrepo_suse'`  |  `'yumrepo_suse'` for all versions |
```

Comment:
this isn't strictly true: `agent5_yumrepo` and `agent5_yumrepo_suse` exist in 4.x as well, and could be mentioned here